comp/core/flare/helpers/perm_info_win.go

```diff
@@ -8,13 +8,114 @@
 
 package helpers
 
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+	"golang.org/x/sys/windows"
+)
+
 // filePermsInfo represents file rights on windows.
 type filePermsInfo struct{}
 
-func (p permissionsInfos) add(filePath string) error {
-	return nil
+func runCmd(cmd *exec.Cmd) string {
+	stdout := bytes.Buffer{}
+	stderr := bytes.Buffer{}
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	err := cmd.Run()
```

Comment:
This function can use [CombinedOutput()](https://pkg.go.dev/os/exec#Cmd.CombinedOutput).